acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/node/PantheonNode.java

```diff
@@ -239,6 +240,15 @@ private String getDiscoveryPort() {
     }
   }
 
+  @Override
+  public Optional<Integer> getJsonRpcHttpPort() {
```

Comment:
How is the JsonRpcHttpPort optional? ...there's always a Json RPC port when JSON-RPC is enabled. Otherwise `null` seems like a more appropriate return value _i.e. dumb programmer problem_

Reply:
done. no longer done this way